setup.py

```diff
@@ -39,7 +39,8 @@
 
 ROOT = os.path.realpath(os.path.join(os.path.dirname(sys.modules["__main__"].__file__)))
 
-# Add Sentry to path so we can import distutils
+# add sentry to path so we can import distutils
+# XXX: consequentially, this means sentry must be pip installed with --no-use-pep517
```

Comment:
This was just to make life easier. Wasn’t ever required iirc.

Reply:
If it works without this, we should 100% remove this as it complicates things a bit too much.